pyzpl/__init__.py

```diff
@@ -57,16 +57,19 @@
 import re
 import sys
 import collections
-import operator as op
+import operator
 
 __version__ = '0.1.9'
 
 PY2 = sys.version_info.major < 3
 
-str = unicode if PY2 else str   # noqa
-
-vitems = op.methodcaller('viewitems') if PY2 else op.methodcaller('items')
-
+if PY2:
+    str = unicode
+    vitems = operator.methodcaller('viewitems')
+else:                                     #python 3
+    # str supports necessary decode
+    vitems = operator.methodcaller('items')
+    
```

Comment:
This is just preference, but with two version-specific  changes under V2 and only 1 under V3, I thought this was more straightforward

Reply:
Sure 👍